mindsdb/api/http/namespaces/file.py

```diff
@@ -120,7 +123,7 @@ def on_file(file):
                         "Error getting file info",
                         "Сan't determine remote file size",
                     )
-                if file_size > 1024 * 1024 * 100:
+                if file_size > MAX_FILE_SIZE:
```

Comment:
let also change error message to `f"Upload limit for file is {MAX_FILE_SIZE >> 20}MB"`